src/convertToObject.js

```diff
@@ -12,7 +12,13 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  sourceString.split(';').map(line => line.trim()).filter(x => x.length > 0)
+    .map(property => property.split(':').map(x => x.trim()))
+    .map(element => {resultObject[element[0]] = element[1]});
```

Comment:
1. You don't write anything to a variable, so last `map` method should be replaced with `forEach`.
2. All chained methods called on same level should have same indentation.

```suggestion
  sourceString
    .split(';')
    .map(line => line.trim())
    .filter(x => x.length > 0)
    .map(property => property.split(':').map(x => x.trim()))
    .forEach(element => {resultObject[element[0]] = element[1]});
```